release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3028,6 +3028,69 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.24
+[id="ocp-4-18-24_{context}"]
+=== RHBA-2025:15714 - {product-title} {product-version}.24 bug fix update
+
+Issued: 17 September 2025
+
+{product-title} release {product-version}.24 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:15714[RHBA-2025:15714] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:15712[RHBA-2025:15712] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.24 --pullspecs
+----
+
+[id="ocp-4-18-24-enhancements_{context}"]
+==== Enhancements
+
+* With this update, the `cluster-etcd-operator` in {product-title} platforms is proactive, modifying the `etcdDatabaseQuotaLowSpace alert`. It triggers at many levels (information, warning, critical) when the etcd quota usage approaches 95%. This enhancement provides cluster administrators with time to address potential issues before the API server is impacted, resulting in a more stable and manageable environment. (link:https://issues.redhat.com/browse/OCPBUGS-61235[OCPBUGS-61235])
+
+*  The name of the `MachineOSConfig` object that was used with {op-system-first} image layering must now be the same as the machine configuration pool where the custom layered image is deployed. Previously, you could use any name. This change was made to prevent attempts to use many `MachineOSConfig` objects with each machine configuration pool. For more information, see xref:../machine_configuration/mco-coreos-layering.adoc#mco-coreos-layering[Image mode for OpenShift].
```

Comment:
```suggestion
* The name of the `MachineOSConfig` object that was used with {op-system-first} image layering must now be the same as the machine configuration pool where the custom layered image is deployed. Previously, you could use any name. This change was made to prevent attempts to use many `MachineOSConfig` objects with each machine configuration pool. For more information, see xref:../machine_configuration/mco-coreos-layering.adoc#mco-coreos-layering[Image mode for OpenShift].
```